CHANGELOG.md

```diff
@@ -17,7 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add Talent Chooser Building Block APIs documentation. [#528](https://github.com/rokwire/rokwire-building-blocks-api/issues/528)
 - Add flags isVirtual and displayOnlyWithSuperEvent to events schema. [#542](https://github.com/rokwire/rokwire-building-blocks-api/issues/542), [#541](https://github.com/rokwire/rokwire-building-blocks-api/issues/541)
 - Add `startDateLimit` query parameter to events search endpoint. [#545](https://github.com/rokwire/rokwire-building-blocks-api/issues/545)
-
+- Add multiple client id to auth library. [#548](https://github.com/rokwire/rokwire-building-blocks-api/issues/548)
```

Comment:
This needs to be moved to the updated Unreleased section. Thanks.

Reply:
What does that mean? It is under unreleased, is there other changelog or something?

Reply:
oh, maybe it is because the new 1.7.0 got merged?

Reply:
yes it was because of the merge. Fixed it.

Reply:
Thanks.